clients/fides-js/src/components/fides.css

```diff
@@ -318,15 +318,26 @@ div.fides-modal-content {
 }
 
 /*Fides Embed*/
+div#fides-embed-container:focus-visible,
+div#fides-embed-container div#fides-modal:focus-visible {
+  outline: none;
+}
+
+div#fides-embed-container div#fides-modal .fides-modal-footer {
+  position: inherit;
+}
+
 div#fides-embed-container .fides-modal-container,
 div#fides-embed-container .fides-modal-overlay {
   position: initial;
-  display: inline;
+  background-color: white;
```

Comment:
Why `white` here? I think the existing background colour is probably brandable with one of the `--fides-*` CSS variables so forcing the background like this would be surprising